README.md

```diff
@@ -109,6 +109,11 @@ System.config({
 
 Plugins handle alternative loading scenarios, including loading assets such as CSS or images, and providing custom transpilation scenarios.
 
+#### Usage
+
+1. First, you should install SystemJS plugin by issuing command `jspm install plugin-name`. For example, you are going to use **CSS** plugin to load css files, then you install it with `jspm install css`.
```

Comment:
@Zoltu Yes, the first step is about JSPM, but becuase it's a requirement to run step 2. So, i think it's relevant here. 

And, thanks for all the suggestion :).